lib/modules/actions.js

```diff
@@ -8,6 +8,8 @@ export const SET_SHELL = 'PLATFORM__SET_SHELL';
 export const REDIRECT = 'PLATFORM__REDIRECT';
 export const REROUTE_PAGE = 'PLATFORM__REROUTE_PAGE'; // re-run the handlers for the page
 
+export const NO_REFERRER  = 'NO_REFERRER';
```

Comment:
I think this is overkill (taking into account reddit/reddit-mobile#938). It's perfectly fine to have an empty referrer field.